src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/RazorMapToDocumentEditsParams.cs

```diff
@@ -2,21 +2,28 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System;
+using System.Runtime.Serialization;
 using Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
 {
+    // Note: This type should be kept in sync with the one in Razor.HtmlCSharp assembly.
     internal class RazorMapToDocumentEditsParams
```

Comment:
DataContract?

Reply:
And this is what happens when I try to open PRs before I'm fully awake. 😅 thanks for catching these!